### PR TITLE
plugin-archive contribution

### PR DESCRIPTION
**Pull request by mdouchement**
This is a plugin to compress or uncompress an archive
Dependency: `github.com/mholt/archiver`

@ovh/cds

### CODE REVIEW

File: contrib/grpcplugins/action/archive/main.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+type archiveActionPlugin struct {
+	actionplugin.Common
+}
+
+func (actPlugin *archiveActionPlugin) Manifest(ctx context.Context, _ *empty.Empty) (*actionplugin.ActionPluginManifest, error) {
+	return &actionplugin.ActionPluginManifest{
+		Name:        "plugin-archive",
+		Author:      "mdouchement <https://github.com/mdouchement>",
+		Description: `This is a plugin to compress or uncompress an archive`,
```
Comment (yesnault) [inline contrib/grpcplugins/action/archive/main.go]:
```suggestion
		Description: `This is a plugin to compress or uncompress an archive. Supported formats: .zip, .tar, .tar.gz, .tar.bz2, .tar.xz, .tar.lz4, .tar.sz, and .rar`,
```

File: docs/content/workflows/pipelines/actions/plugins/plugin-archive.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## Parameters
+
+* **source**: The source path
+* **destination**: The destination path
+* **action**: `compress` or `uncompress` an archive
+
+
+## More
+
+More documentation on [Github](https://github.com/ovh/cds/tree/master/contrib/grpcplugins/action/archive/README.md)
```
Comment (yesnault) [inline docs/content/workflows/pipelines/actions/plugins/plugin-archive.md]: this file does not exist, you can remove all ##more section I think

File: contrib/grpcplugins/action/archive/main.go [hunk 3]
```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/mholt/archiver"
```
Comment (richardlt) [inline contrib/grpcplugins/action/archive/main.go]: The dependency is not in cds vendors, this can't compile

Comment (richardlt) [reply contrib/grpcplugins/action/archive/main.go]: We will add it after the merge because of the go 1.11 PR

Comment (mdouchement) [reply contrib/grpcplugins/action/archive/main.go]: Yes I didn't add it in the vendors.

**Pull request merged**